### DISCUSSION TITLE
Video tutorial

### DISCUSSION THREAD

**Discussion by satyaidk** [is-a-dev/register#8081] [Q&A] [answered]
Can I get the video tutorial on this setting up my is-a.dev domain for better understanding to setup my domain is this available.
I'm little bit confusing, so that's why asking 😕

**Comment by wdhdev** [comment, marked as answer]
I have gone and made a simple tutorial and uploaded it to YouTube just for you!
https://www.youtube.com/watch?v=fwsD0QFUTqs

**Reply by satyaidk** [reply]
Thank You 😊
